app/assets/stylesheets/components/_tag.scss

```diff
@@ -2,7 +2,7 @@
   color: white;
   border-radius: 8px;
   /*Background color in show page directly linked with the color in the DB*/
-  /*background-color: purple;*/
```

Comment:
Some variables have a "$" some other not... Why? Just curious.

Reply:
Actually, there should be no more purple (not in our chart) 🤔
+ main colors are built-in, so you don't need a Sass variable for them

Reply:
Like paradize-pink? Nice